package.json

```diff
@@ -35,7 +35,7 @@
     "prepublish": "npm run build"
   },
   "dependencies": {
-    "immutable": "3.7.0"
+    "immutable": ">=3.8.1"
```

Comment:
I really dislike `peerDependencies` as a rule. I understand why it works the way it does, but I feel like it is so conservative as to require a lot of manual work. I generally feel `>=` gets the right answer more often.

I could be persuaded to decrease the minimum version to 3.7.6, though. That would cause us to use whatever version is installed by a project's other things as long as it is at least 3.7.6, and then it will install its own otherwise.